features/ContentEdit/create_without_draft.feature

```diff
@@ -12,3 +12,12 @@ Scenario: Create a folder without a draft
      When I fill in the folder edit form
       And I press "Publish"
      Then I am on the View of the Content that was published
+
+Scenario: Create a folder without a draft
```

Comment:
same name as in the previous scenario, I guess it should changed to something more relevant?